src/AppMain.css

```diff
@@ -0,0 +1,140 @@
+@import url('https://fonts.googleapis.com/css2?family=Mulish:wght@400;700;800&display=swap');
+
+div {
+  background: #EEF0F1;
+}
+
+.App {
+  text-align: center;
+  font-family: 'Mulish', sans-serif;
+  min-height: 50rem;
+}
+
+.NavBar {
+  width: 100%;
+  height: 5rem;
+  text-align: center;
+  display: flex;
+  justify-content: left;
+  align-items: center;
+  flex-direction: row;
+  position: fixed;
+  top: 0;
+  background: #4267B2;
+  color: white;
+  min-width: 50%;
+  white-space: nowrap;
+  font-weight: 800;
+  z-index: 1;
+}
+
+#TeachLA {
+  width: 3.75rem;
+  height: 3.75rem;
+  left: 20px;
+  margin: 1.25rem;
+}
+
+#BiasByUs {
+  font-size: 3rem;
+  margin: 0.5625rem;
+}
+
+#currentCaseStudy {
+  font-size: 1.875rem;
+  margin: 1.25rem;
+  position: relative;
+  top: 0.3125rem;
+}
+
+button {
+  background-color: #EEF0F1;
+  width: 4.063rem;
+  min-width: 4.063rem;
+  height: 4.063rem;
+  border-radius: 50%;
+  border: none;
+  position: relative;
+  margin-left: auto;
+  margin-right: 4.375rem;
+  display: inline-block;
+}
+button:focus {
+  outline: none;
+}
+.triangle {
+  width: 0;
+	height: 0;
+	border-left: 1.25rem solid transparent;
+	border-right: 1.25rem solid transparent;
+  border-top: 1.563rem solid black;
+  position: relative;
+  left: 0.375rem;
+  top: 0.25rem;
+}
+.up {
+  transform: rotate(180deg);
+  top: -0.0625rem;
+}
+
+
+
+
+/* everything below is for component placeholders */
+
+#dropdownPlaceholder {
+  font-size: 1.5rem;
+  background-color: black;
+  color: white; 
+  height: 12.5rem;
+  width: 12.69rem;
+  position: fixed;
+  top: 6.25rem;
+  right: 0;
+}
+
+.Posts {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  position: relative;
+  top: 5rem;
+  z-index: 0;
+}
+p {
+  font-size: 2rem;
+  border: black 0.0625rem solid;
+  /*width: 59.38rem;*/
+  width: 66%;
+  height: 25rem;
+  /*padding: 400px 300px 400px 300px;*/
+  background-color: white;
+  border-radius: 1.25rem;
+}
```

Comment:
Similar comment on adjusting `p`; use another class instead.

Also, are you sure you want to manually set the `height` here?

Reply:
Since Matthew wrote the code for the Post components (which I don't have access to yet), I decided to create sample 'posts' at the center of the page so that I can visualize how the app would generally look. I will be replacing most of the styling here (including the manually set height) with the styling that comes with Matthew's Post component.
![image](https://user-images.githubusercontent.com/50722281/93561260-b5b14580-f938-11ea-91ce-1a0f1bc142b8.png)
This is how the page looks on my end right now.